config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Rubygamedev::Application.routes.draw do
-
   get '/auth/:provider/callback', to: 'sessions#create_from_omniauth'
   get '/auth/failure', to: 'sessions#auth_failure'
 
   resources :users, only: [:create, :edit, :update]
   get '/account' => 'users#edit_current_user', as: :account
 
+  resources :posts
+  resources :internal_posts, controller: "posts", type: "InternalPost"
```

Comment:
I think we can remove these routes. We don't need to have different routes for internal/external posts.